public/css/index.css

```diff
@@ -238,149 +177,60 @@ article .post-panel .row .post-container .selectedImage .image-container .close-
 }
 
-article .post .row .poster-photo {
-    height: 80px;
-    width: 80px;
-    background-color: white;
-    margin: 7px auto 0 auto;
+article .post .row .poster-information {
+    width: calc(100%/6);
 }
 
-article .post .row .poster-photo:hover {
-    opacity: 1;
+article .post .row .poster-information .poster-photo {
+    height: 9.5%;
 }
 
-article .post .row .poster-name {
-    color: white;
-    font-size: 0.5em;
-    margin: 5px auto 0 auto;
-    background-color: var(--gray);
-    width: 80px;
-    height: 20px;
-    line-height: 20px;
-    font-weight: bold;
+article .post .row .poster-information .poster-name {
+    font-size: 10px;
 }
 
-article .post .row .poster-name:hover {
-    opacity: 0.8;
+article .post .row .post-content-panel {
+    width: calc(100%*5/6);
 }
 
-article .post .row .post-content {
-    background-color: white;
-    margin: 7px 0 10px 0;
+article .post .row .post-content-panel .post-content {
     width: calc(100% - 40px);
-    height: auto;
-    padding: 6px 6px 12px 12px;
-    position: relative;
 }
 
 /*隱藏scrollbar*/
 article::-webkit-scrollbar {
     display: none;
 }
 
-article .post .row .post-content .likes-icon {
-    width: 20px;
-    padding-top: 4px;
+article .post .row .post-content-panel .post-content .likes-icon {
     color: var(--navy-blue);
 }
 
-article .post .row .post-content .likes-quantity {
-    width: 100px;
-    letter-spacing: 2px;
-}
-
-article .post .row .post-content p {
-    word-wrap: break-word;
-}
-
-article .post .row .post-content img {
-    border-radius: 0;
-}
-
-article .post .row .post-content .feedback-panel {
-    width: 96.5%;
-    margin-top: 10px;
-}
-
-article .post .row .post-content .feedback-panel .interaction-bar {
-    width: 40%;
-}
-
-article .post .row .post-content .feedback-panel .interaction-bar button {
-    background-color: var(--gray);
-    width: 80px;
-    height: 30px;
-}
-
 aside {
-    position: fixed;
     right: 1%;
     top: 95px;
-    height: auto;
-    width: 33%;
-}
-
-aside ul {
-    padding-left: 0;
 }
 
 aside ul .side-bar {
-    background-color: white;
-    height: 80px;
     transition: height 0.5s, line-height 0.5s;
     transition-delay: 0.08s;
-    margin-bottom: 16px;
     border-radius: 8px;
 }
 
-aside ul .side-bar {
-    font-weight: 900;
-    font-size: 1.3em;
-
-}
-
 aside ul .side-bar:hover {
-    height: 242px;
+    height: 236px;
 }
 
 aside ul .side-bar .bar-name {
     line-height: 80px;
-    margin-bottom: 0;
-}
-
-aside ul .side-bar ul {
-    border-top: 3px solid var(--light-gray);
-    width: 94%;
-    margin-left: auto;
-    margin-right: auto;
 }
 
 aside ul .side-bar ul li .explore-item {
-    display: inline-block;
-    height: 40px;
-    width: 100%;
-    margin-top: 10px;
     line-height: 40px;
 } 
-
-aside ul .side-bar ul li .explore-item:hover {
-    background-color: var(--light-gray);
-}
-
 aside .site-information {
-    color: white;
     bottom: 0;
-    font-weight: 100;
-    letter-spacing: 1px;
 }
```

Comment:
Add a new line here.

You're doing a great job refactoring codes from Bootstrap to Tailwind. There're still several classes to remove inside `index.css` to use Tailwind classes directly.

Keep processing this! You're almost there :))))